### PR TITLE
Categorías de sesiones [x2.1.3.6]a

### PR DESCRIPTION
**Pull request by ilosada**

### CODE REVIEW

File: tests/scripts/insert_sessions_categories.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
```
Comment (jmontoyaa) [inline tests/scripts/insert_sessions_categories.php]: License

File: tests/scripts/insert_sessions_categories.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+/**
+ * This script generates four session categories.
+ */
+require_once '../../main/inc/global.inc.php';
+api_protect_admin_script();
+$categories = array(
```
Comment (jmontoyaa) [inline tests/scripts/insert_sessions_categories.php]: Don't use Spanish

Comment (ywarnier) [inline tests/scripts/insert_sessions_categories.php]: This is a short example script (we needed these categories names specifically). Given it is in tests/scripts/ I don't think the rules with content name really need to apply. The rest still does (license, security, formatting, variables names, etc). Just that the content is specific.

Comment (jmontoyaa) [inline tests/scripts/insert_sessions_categories.php]: MMMmmm ok. Good to know!

**Pull request merged**